pom.xml

```diff
@@ -45,10 +45,11 @@
     <dagger.version>0.9</dagger.version>
     <otto.version>1.3.1</otto.version>
     <gson.version>2.2</gson.version>
-    <junit.version>4.10</junit.version>
+    <junit.version>4.11</junit.version>
```

Comment:
It should be reflected in its dependencies. We happen to be overriding it with an old version which Maven does wrong.